src/cl_sii/dte/data_models.py

```diff
@@ -165,21 +162,17 @@ def slug(self) -> str:
     # Validators
     ###########################################################################
 
-    @pydantic.v1.validator('folio')
+    @pydantic.field_validator('folio')
```

Comment:
⚠️ This review comment applies to all similar cases.

Add `classmethod` decorator to Pydantic field validators.

See [Pydantic documentation](https://docs.pydantic.dev/2.4/concepts/validators/#field-validators):

> `@field_validators` are "class methods", so the first argument value they receive is the `UserModel` class, not an instance of `UserModel`. **We recommend you use the `@classmethod` decorator on them below the `@field_validator` decorator to get proper type checking**.
> [emphasis added]

```suggestion
    @pydantic.field_validator('folio')
    @classmethod
```